src/main/java/com/softwire/todos/TodoCheckerMain.java

```diff
@@ -90,10 +93,15 @@
                     "you have multiple jobs running against different codebases but with the same JIRA project, " +
                     "otherwise the jobs will interfere with each other.")
     public String jobName = null;
+
+    @Option(name = "--report-file",
+            usage = "Write report of errors to this file as well as to the console.")
+    public String reportFile = null;

```

Comment:
This feels a bit redundant; can't users achieve the same with "[`tee`](http://man7.org/linux/man-pages/man1/tee.1.html)"? Fine by me to leave if you think it is useful.

Reply:
You can, but you also have to restrict to error logs and strip off the log message prefix.  I'm running it against a codebase with loads of preexisting TODOs, so a report is useful while we get the numbers down, and this was the simplest solution.  Don't mind if you want to remove it, up to you.

Reply:
Leave it in, if it's useful to you